installing/install_config/enable-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,16 @@
+[id="enable-cluster-wide-proxy"]
+= Configuring cluster-wide proxy
+include::modules/common-attributes.adoc[]
+:context: enable-cluster-wide-proxy
+
+toc::[]
+
+Production environments can deny direct access to the Internet and instead have
+an HTTP or HTTPS proxy available. Configuring {product-title} to use these
+proxies can be as simple as setting standard environment variables in
```

Comment:
s/these proxies/a cluster-wide proxy/

Reply:
@bergerhoffer this is not completely accurate:
```
Configuring {product-title} to use these proxies can be as simple as setting standard environment variables in configuration or JSON files.
```
I would change to something like:
```
Configuring {product-title} to use these proxies can be as simple as modifying the `Proxy` object for existing clusters or configuring proxy settings in `install-config.yaml` for new clusters.
```